src/assets/translations/fr/main.json

```diff
@@ -348,8 +348,8 @@
     "clickToConnect": "Cliquez pour connecter un porte-monnaie"
   },
   "transactionRow": {
-    "send": "%{symbol} envoyés",
-    "receive": "%{symbol} reçus",
+    "send": "envoyé",
```

Comment:
@firebomb1 FYI now that the symbol is removed, reverted these two

Reply:
@gomesalexandre These should start with a capital letter then, to address the underlying issue. Same in German for "gesendet" -> "Gesendet".

Reply:
Nice one! c837efe980ef5a0412f0c0075c16088c63607e86